crates/ruff_python_ast/src/helpers.rs

```diff
@@ -961,9 +967,15 @@ where
             | Stmt::AsyncFunctionDef(_)
             | Stmt::Try(_)
             | Stmt::TryStar(_) => {}
-            Stmt::If(ast::StmtIf { body, orelse, .. }) => {
+            Stmt::If(ast::StmtIf {
+                body,
+                elif_else_clauses,
+                ..
+            }) => {
                 walk_body(self, body);
-                walk_body(self, orelse);
+                for clause in elif_else_clauses {
+                    walk_body(self, &clause.body);
+                }
```

Comment:
Nit: I explained this in your other PR. I think we should change `Visitor` to have instead a `visit_elif_else_clause` method to be consistent with `ExceptHandler` (it has a visit function for every node).  We then need to call `walk_elif_else_clause` here and override the `visit_elif_else_clause`
```suggestion
                for clause in elif_else_clauses {
                	walk_elif_else_clause(&clause);
                }
```